services/ui-src/src/types/index.ts

```diff
@@ -69,12 +69,12 @@ export enum ReportStatus {
 
 // REPORT PROVIDER/CONTEXT
 
-export interface ReportDetails {
+export interface ReportKeys {
   state: string;
   reportId: string;
```

Comment:
Hmm interesting idea. I'm not opposed! I think `selectedReportId` still makes sense, because it's not the selected reportId, it's the selected report's id. 🤷 I'm good with changing it.